Services/OpencastService.php

```diff
@@ -117,7 +117,7 @@ public function generateSbsTrack(MultimediaObject $multimediaObject, $opencastUr
         }
 
         $track = $tracks[0];
-        $path = $this->getPath($track->getUrl());
+        $path = $this->getPath($track->getPath());
```

Comment:
Was a bug on generateSbsTrack getPath() ?

Reply:
Yep, there was an issue with getUrl:

TTK-21532 OpencastBundle: Fixed SBS op erroring for edge case.
    The edge case only happens now that we can use a mmobj as an import template,
    when there's a mmobj with manually uploaded tracks that is used as template,
    the operation does not have a valid 'path', since we call 'getUrl'.
    Solved by calling 'getPath' instead.